lib/kubernetes-deploy/runner.rb

```diff
@@ -19,11 +19,11 @@ def self.with_friendly_errors
       exit 1
     end
 
-    def initialize(namespace:, environment:, current_sha:, template_folder: nil, context:)
+    def initialize(namespace:, environment:, current_sha:, context:, template_dir: nil)
       @namespace = namespace
       @context = context
       @current_sha = current_sha
-      @template_path = File.expand_path('./' + (template_folder || "config/deploy/#{environment}"))
+      @template_dir = template_dir || File.expand_path("./config/deploy/#{environment}")
```

Comment:
Doesn't this change `ENV['K8S_TEMPLATE_FOLDER']` from being relative to being absolute? When this is being run in shipit, the current directory will always be app root, which is the reference point devs setting this option have access to. They won't know the absolute path.